docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: '3.8'
+services:
+
+  postgres:
+    image: postgres:alpine
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+    environment:
+        POSTGRES_PASSWORD: showtime
+        POSTGRES_USER: showtime
+        POSTGRES_DB: showtime
+
+  brave:
+    image: registry.comp.ystv.co.uk/brave:latest
+    ports:
+      - "5000:5000"
+
+  nginx:
+    image: registry.comp.ystv.co.uk/nginx/nginx-rtmp:latest
```

Comment:
Outside the scope of this ticket, but if we do refactor how we make this image is built into a public repo, would be good to drop a link as well